### PR TITLE
Verbose logging for TN Scale

### PR DESCRIPTION
**Pull request by sonicaj**
This PR introduces changes where we make TN Scale logs more verbose wrt retrieving disks and creating pools.

### CODE REVIEW

**Comment by sonicaj** [comment]
For disks, we are seeing a case where potentially the disk name is a malformed string which shouldn't be possible ideally. Because of this, when we try to do a low level probe, it fails. For pool creation we are getting a random error `No such pool or dataset`. I am unable to reproduce them so far but they do happen once in a while, so this is to get more insight in the context of the issue.

**Comment by william-gr** [comment]
Perhaps we should only log this in case create fails?

**Comment by sonicaj** [comment]
That’s a very good idea. I’ll make amends

File: src/middlewared/middlewared/plugins/pool.py [hunk 1]
```diff
@@ -670,6 +671,10 @@ def pool_extend(self, pool):
             )
         except Exception as e:
             # Something wrong happened, we need to rollback and destroy pool.
+            if IS_LINUX:
+                self.middleware.logger.debug(
+                    'Pool %s is being created with topology %s', data['name'], data['topology'],
```
Comment (william-gr) [inline src/middlewared/middlewared/plugins/pool.py]: You mean "pool %s failed to create..." ?

**Pull request merged**